### ISSUE TITLE
CapsLock as a VoiceOver key does not work when Karabiner Elements is running

### ISSUE THREAD

**Issue by ushiushix** [tekezo/Karabiner-Elements#698]
When Karabiner Element is running, CapsLock key as VoiceOver key is ignored.
Isn't it possible to pass any key unfiltered?

VoiceOver is a built-in screen reader for macOS. It can launch and stop with command+F5 toggle.
VoiceOver provides many keystroke for screen reading with a modifier combination called VoiceOver key. It is control+option by default. For example, VoiceOver key + left or right arrow key moves the VoiceOver focus to the previous or next element on the screen.

VoiceOver preference can be accessed by VoiceOver key + F8 with VoiceOver running, or System preference -> Accessibility -> VoiceOver -> VoiceOver utility. Here you can choose VoiceOver key from control+option, CapsLock, or both.
Setting VO key to CapsLock benefits because control+option has potential conflicts with many applications, and single keystroke is simply better than double.

However, when Karabiner Elements is running, CapsLock does not work as VoiceOver key even if it is selected in the VoiceOver preference.
I can imagine that VoiceOver is hooking key events and which can conflict with keyboard remapping, but hope it can work together.

Step to reproduce:
0. Quit Karabiner elements.
1. Set VO key to both CapsLock and control+option from VoiceOver preference.
2. Enable VoiceOver.
3. Press CapsLock + d to move focus to the dock for example. You can also try control+option+d to see both works as the same.
4. Run Karabiner Elements. No configuration is required.
5. Do the step 3. again. Here you see nothing happens with CapsLock+d.

Note:
- With VoiceOver key set to only control+option, VO says "CapsLock on" when the Caps key is pressed. But when VO key is set to CapsLock it says nothing even when Karabiner Elements is running. It means that VoiceOver knows that CapsLock as VO key is pressed but it fails using it as the VO key. Weird.
- In Karabiner Event viewer, CapsLock key is recognized as CapsLock even if VoiceOver is running.

Environment:
- MacBook Air 2013 with built-in JIS keyboard
- macOS Sierra 10.12.5 Beta1(Also in 10.12.4)
- Karabiner Elements 0.90.92

**Comment by gurix** [comment]
Looks like the same issue we have on https://github.com/keycastr/keycastr/issues/100#issuecomment-350301805

**Comment by ushiushix** [comment]
Only to record the issue,
Now in Karabiner 12.0.0 and macOS 10.13.4, CapsLock key is assignable to VoiceOver key while using Karabiner-Elements.
Could be an effect of virtual keyboard improvements.
Thanks.
